.github/workflows/publishDockerToGhcr.yml

```
@@ -0,0 +1,41 @@
+name: Create and publish a Docker image
```

I'm not sure, but maybe we can just use more generic "container" image :) 
```suggestion
name: Create and publish a container image
```